compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -351,9 +355,16 @@ impl Inliner<'tcx> {
         if callee_body.basic_blocks().len() <= 3 {
             threshold += threshold / 4;
         }
+
+        // Give a huge bonus to local, unexported functions called only once in current module.
+        if let Some(callee_def_id) = callsite.callee.def_id().as_local() {
+            if (self.tcx.mir_inliner_crate_fn_unexported_callees_count(callee_def_id)) == Some(1) {
+                threshold *= 2;
```

Comment:
Does this factor need to be tuned somehow?
cc @wesleywiser

Reply:
I suspect we will probably need to tune it yeah, but it also seems ok to land as-is since this pass isn't enabled by default.

Reply:
Factors currently are just with gut feeling. Feel free to suggest better or well known values.